examples/java/src/main/java/com/xebia/functional/xef/java/auto/util/Util.java

```diff
@@ -0,0 +1,22 @@
+package com.xebia.functional.xef.java.auto.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Util {
+
+    private static final BufferedReader sysin = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
Both BufferedReader and InputStreamReader implement AutoCloseable and should be specified in try-with-resources blocks so they can be properly finalized and closed when done.

Reply:
Fixed. Added Autoclosable interface and renamed to ConsoleUtil.java